librz/analysis/arch/ppc/ppc_il.h

```diff
@@ -23,15 +23,6 @@
  */
 #define DCACHE_LINE_SIZE 128
 
-/**
```

Comment:
Why this? Is it unused? Please send as a separate PR then please, so we can remove it from this PR